docs/skills/choice.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: guide
+---
+# Installing 
+
+The choice skill can be installed from the npm registry.
+
+```bash 
+# using npm
+npm i @botpress/skill-choice
+# using yarn 
+yarn add @botpress/skill-choice
+```
+
+Once installed, restart your bot and the skill will be automatically detected and loaded by your bot.
+
+# Using the choices skill
+
+Skills are added to the flow of your bot using the Botpress Flows GUI.
+
+You will find a dropdown list in the toolbar to `Insert Skill` 
+
+![Image showing the Skills Menu][Skills_Menu]
+
+After selecting the choice skill you will need to click on the folder icon to ether create or select an existing choice configuration.
+
+When you chick on the `Create new Single Choice` you will be presented with a form where you will be able to configure this instance.
+
+- Message - This is the Question that is asked to the user
+- Choices - This is a collection of the buttons that will be presented to the user
+  - Message - This is the text that will be shown in the button
+  - Value - This is the value the bot will receive when the corresponding button is pressed
+
+![GIF showing how to create a new choice][Choice_Gif]
+
+# Advanced
+
+## Hiding Choices
+
+As of version 10.30, you are able to configure the choice skill to handle a choice but stop it from being displayed to the user in the conversation. This allows you top handle edge cases or unpopular conversation paths without cluttering up the user interface for most users.
```

Comment:
You can add a link to changelog of 10.30 here: https://github.com/botpress/botpress/blob/master/CHANGELOG.md#10300-2018-08-08

Also changelog mentions this change for 10.31.0 not 10.30.0